prov/efa/test/efa_unit_test_mocks.c

```diff
@@ -5,68 +5,128 @@
 #include <setjmp.h>
 #include <cmocka.h>
 #include "efa.h"
+#include "efa_unit_test_mocks.h"
 
-void __real_rxr_pkt_handle_send_completion(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry);
+/* mock of rdma-core functions */
+uint8_t g_ibv_gid_mock[EFA_GID_LEN] = {0xfe, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x10, 0xcb, 0x06, 0xff, 0xfe, 0x66, 0x83, 0x2f};
```

Comment:
Does the value of this array have any significance? A comment will be helpful.

Reply:
Removed this array, and use self address instead.